apps/btc_family/ltc/ltc_app.c

```diff
@@ -0,0 +1,143 @@
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ *
+ * "Commons Clause" License Condition v1.0
+ *
+ * The Software is provided to you by the Licensor under the License,
+ * as defined below, subject to the following condition.
+ *
+ * Without limiting other conditions in the License, the grant of
+ * rights under the License will not include, and the License does not
+ * grant to you, the right to Sell the Software.
+ *
+ * For purposes of the foregoing, "Sell" means practicing any or all
+ * of the rights granted to you under the License to provide to third
+ * parties, for a fee or other consideration (including without
+ * limitation fees for hosting or consulting/ support services related
+ * to the Software), a product or service whose value derives, entirely
+ * or substantially, from the functionality of the Software. Any license
+ * notice or attribution required by the License must also include
+ * this Commons Clause License Condition notice.
+ *
+ * Software: All X1Wallet associated files.
+ * License: MIT
+ * Licensor: HODL TECH PTE LTD
+ *
+ ******************************************************************************
+ */
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include "ltc_app.h"
+
+#include "btc_helpers.h"
+
+/*****************************************************************************
+ * EXTERN VARIABLES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * PRIVATE MACROS AND DEFINES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * PRIVATE TYPEDEFS
+ *****************************************************************************/
+
+/*****************************************************************************
+ * STATIC FUNCTION PROTOTYPES
+ *****************************************************************************/
+
+/**
+ * @brief Check if the purpose index is supported by the Litecoin app.
+ *
+ * @param purpose_index The purpose index to be checked
+ *
+ * @return bool Indicates if the provided purpose index is supported
+ * @retval true The provided purpose index is supported
+ * @retval false The provided purpose index is not supported
+ */
+static bool is_purpose_supported(uint32_t purpose_index);
+
+/*****************************************************************************
+ * STATIC VARIABLES
+ *****************************************************************************/
+
+/**
+ * Litecoin supports Segwit and Native-Segwit but not Taproot unlike Bitcoin
+ * Values can be verified from the following references:
+ * https://github.com/litecoin-project/litecoin
```

Comment:
i couldn't find anything here. is this reference useful?

Reply:
[the chainparams](https://github.com/litecoin-project/litecoin/blob/master/src/chainparams.cpp) have some of the info

Reply:
oh!

in that case, [use this link](https://github.com/litecoin-project/litecoin/blob/5ac781487cc9589131437b23c69829f04002b97e/src/chainparams.cpp#L65-L177)

Reply:
done